src/app/ongs/[slug]/templates/common/sections/faq/question-item/index.tsx

```
@@ -0,0 +1,53 @@
+            }`}
+          />
+        </figure>
+      </div>
+
+      {isClient && (
+        <p
+          style={{
+            maxHeight: isQuestionOpen ? '400px' : '0px'
+          }}
+          className={`overflow-hidden text-left text-sm text-slate-500 transition-all duration-300 ease-in-out md:text-base`}
+          dangerouslySetInnerHTML={{ __html: copy.description }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)